src/App.js

```diff
@@ -25,8 +25,8 @@ const App = () => {
     <>
       <Header />
       <Routes>
-        <Route path='/' element={<Home />} />
-        <Route path='/botindex' element={<BotIndex />} />
+        <Route path="/" element={<Home />} />
+        <Route path="/botindex" element={<BotIndex />} />
```

Comment:
I would be consistent with the use of single or double quotes here. If someone on your team has autoformatting it would be worth agreeing on a style guide.